kernel/packages/shared/types.ts

```diff
@@ -245,6 +259,18 @@ export interface ILand {
   mappingsResponse: MappingsResponse
 }
 
+export interface IPortableExperience {
+  /**
+   * sceneId: Now it is either an internal identifier or the rootCID.
+   * In the future will change to the sceneCID
```

Comment:
I'm not sure I get this. Could you explain it a little bit?

Reply:
done